webpack.config.js

```diff
@@ -215,13 +216,17 @@ const config = (module.exports = {
 });
 
 if (WEBPACK_BUNDLE === "hot") {
+
+  const localIPAddress = getLocalIPAddress("IPv4") || getLocalIPAddress("IPv6") || "0.0.0.0";
```

Comment:
Question: Are we using full caps for abbreviation or just the first letter? From my observation, it seems to be the latest, but I could be wrong. e.g. one of the example of abbreviation https://github.com/metabase/metabase/blob/9e2918d1188a555f303bed3be230841019781a42/frontend/src/metabase/lib/settings.ts#L171

Reply:
Good catch, I've adjusted it accordingly.